demos/python_demos/llm_text_generation/utils.py

```diff
@@ -20,11 +20,6 @@
 
 def serialize_prompts(prompts):
     infer_input = grpcclient.InferInput("pre_prompt", [len(prompts)], "BYTES")
-    if len(prompts) == 1:
-        # Single batch serialized directly as bytes
-        infer_input._raw_content = prompts[0].encode()
-        return infer_input
-    # Multi batch serialized in tritonclient 4byte len format
     infer_input._raw_content = serialize_byte_tensor(
-        np.array(prompts, dtype=np.object_)).item()
+        np.array(prompts, dtype=np.bytes_)).item()
```

Comment:
Why do you change it? Is that not a problem? https://github.com/triton-inference-server/client/blob/8f8ae561bf99def7c5561ad907ea9309db99ec30/README.md?plain=1#L602-L609